source/cli.js

```diff
@@ -72,6 +72,8 @@ updateNotifier({pkg: cli.pkg}).notify();
 (async () => {
 	const pkg = util.readPkg();
 
+	util.checkIgnoreStrategy(pkg);
```

Comment:
I think that this should be moved to `index.js` and run before all of the tasks, though I'd like to know what others think as well.

Reply:
It seems like that wouldn't work. I've tried moving it there, but `index.js` exports this function that's only executed after the user answer the prompts. IMO it's better to show the warning earlier.

Reply:
Why would putting it [here](https://github.com/cironunes/np/blob/1003fabb05c82676e51f9160422ff579ec196615/source/index.js#L47) not work?
That would still show the warning at the same place, and keep `np`'s behavior consistent even when using it programmatically.

Reply:
It won't work because it [awaits for the ui](https://github.com/cironunes/np/blob/1003fabb05c82676e51f9160422ff579ec196615/source/cli.js#L100) and then run [index](https://github.com/cironunes/np/blob/1003fabb05c82676e51f9160422ff579ec196615/source/cli.js#L107). That's why I asked if we should move it to ui in the first place.

I updated my PR to do that. This way we keep it consistent between using the CLI and programatically.

Reply:
Can you send a screenshot of how everything currently looks?

Reply:
That's how it's looking like:
![Screen Shot 2019-06-06 at 8 59 58 am](https://user-images.githubusercontent.com/469908/58995983-b2747200-8839-11e9-8ab1-51d936f047af.png)

That's another take on it:
![Screen Shot 2019-06-06 at 9 01 13 am](https://user-images.githubusercontent.com/469908/58995984-b2747200-8839-11e9-9ade-a9424584b65a.png)

Reply:
I like the second option better. Also, the error message should contain a short explanation on why the `files` property / `.npmignore` should be set when publishing a package (e.g. `setting one of those will prevent you from publishing development-specific files along with your package's source code to npm`).

Reply:
Updated!